packages/react-dom-bindings/src/client/ReactDOMHostConfig.js

```diff
@@ -678,12 +709,36 @@ export function unhideTextInstance(
 }
 
 export function clearContainer(container: Container): void {
-  if (container.nodeType === ELEMENT_NODE) {
-    ((container: any): Element).textContent = '';
-  } else if (container.nodeType === DOCUMENT_NODE) {
-    if (container.documentElement) {
-      // $FlowFixMe[incompatible-call]
-      container.removeChild(container.documentElement);
+  if (enableHostSingletons) {
+    if (container.nodeType === ELEMENT_NODE) {
+      // We have refined the container to Element type
+      const element: Element = (container: any);
+      switch (element.tagName.toLowerCase()) {
```

Comment:
We can assume this is a HTML and not XHTML document and not do the lowercasing. Just compare the upper case. This should be done elsewhere too for Float.

Reply:
got rid of lowercases